src/main/java/org/springframework/integration/aws/support/AwsSendFailureException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.aws.support;
+
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessagingException;
+
+import com.amazonaws.AmazonWebServiceRequest;
+
+/**
+ * An exception that is the payload of an {@code ErrorMessage} when a send fails.
+ *
+ * @author Jacob Severson
+ * @since 1.1.0
+ */
+public class AwsSendFailureException extends MessagingException {
+
+	private static final long serialVersionUID = 1L;
+
+	private final AmazonWebServiceRequest record;
```

Comment:
This is definitely not a `record`, but just `request`.

I like your idea to make this exception as generic as possible. 😄 

Well, not all AWS services call their request execution action as `send`. So, maybe something like `AwsRequestFailureException` ?